packages/shared/ReactTypeOfWork.js

```diff
@@ -37,3 +37,4 @@ export const Fragment = 10;
 export const Mode = 11;
 export const ContextConsumer = 12;
 export const ContextProvider = 13;
+export const TimeoutComponent = 15;
```

Comment:
should be `14` here for consistence and also added to `TypeOfWork` above?

Reply:
It's probably because @acdlite redacted `LoadingComponent` to keep us in suspense until @gaearon's talk 🤫